incident/models/incident_page.py

```diff
@@ -770,6 +770,24 @@ class IncidentPage(MetadataPageMixin, Page):
 
     parent_page_types = ['incident.IncidentIndexPage']
 
+    search_fields = Page.search_fields + [
+        index.SearchField('body'),
+        index.SearchField('city'),
+        index.RelatedFields('state', [
+            index.SearchField('name'),
+        ]),
+        index.SearchField('introduction'),
+        index.SearchField('teaser'),
+        index.RelatedFields('teaser_image', [
+            index.SearchField('attribution'),
+        ]),
+        index.SearchField('image_caption'),
+        index.RelatedFields('updates', [
+            index.SearchField('title'),
+            index.SearchField('body'),
+        ]),
```

Comment:
I see that we are using only the [title and body indexes](https://github.com/freedomofpress/pressfreedomtracker.us/pull/1626/files#diff-a3770fdca8d1cfb14cd8af7251251982012e063185ec18ea1b4573ba2985c9f7R524-R529) in the search. Adding these fields is to future proof?

Reply:
So how Wagtail's `IndexEntry` model works is not super well documented, but the title and body fields on the from your link are its built-in fields that contain searchable data (there is also a third one called `autocomplete` that allows for partial matches -- intended to be used for find-as-you-type varieties of searches). From how they are named, it might seem like there is a 1-to-1 correspondence between those fields and the fields of the model being indexed, but that's only true for `title`. The `body` field (on `IndexEntry`) includes all the search data that's not part of the title -- so all the fields that are listed in the `search_fields` attribute of `IncidentPage` will have their searchable text pulled in and indexed. You can actually test this out by performing a search based on, say, text in the `introduction` field and see that you will get results.